src/sentry/models/avatars/control_base.py

```diff
@@ -22,3 +22,7 @@ def file_fk(cls) -> str:
         if ControlFileBlob._storage_config():
             return "control_file_id"
         return "file_id"
+
+    @property
+    def get_file_id(self):
+        return self.control_file_id
```

Comment:
Should we coalesce to `file_id`? While all the avatars in saas have control_file_id, we won't have that field populated in other environments as they won't have migrated to controlfile yet.